kubernetes/client/models/v1_pod.py

```diff
@@ -12,6 +12,7 @@
 
 import pprint
 import re  # noqa: F401
+from datetime import datetime
```

Comment:
This file is generated using openapi-generator (see the top of this file). Could you send the fix to the upstream?

Reply:
my bad, thank you for clarification. do you mean openapi by upstream or something else?